pyanaconda/bootloader.py

```diff
@@ -1529,17 +1530,12 @@ def write_password_config(self):
         if not self.password and not self.encrypted_password:
             return
 
-        users_file = iutil.getSysroot() + "/etc/grub.d/01_users"
+        users_file = "%s%s/%s" % (iutil.getSysroot(), self.config_dir, _passwd_file)
```

Comment:
You got this pylint error: ``E0602(undefined-variable):1533,71: GRUB2.write_password_config: Undefined variable '_passwd_file'`` so please use ``self._passwd_file`` or better ``self.__class__._passwd_file``.